caps/templates/caps/council_cards/scorecard.html

```diff
@@ -0,0 +1,227 @@
+{% extends "caps/council_cards/base.html" %}
+
+{% load caps_templatetags %}
+{% load humanize %}
+
+{% block title %}
+    Climate Plan Scorecard
+{% endblock title %}
+
+{% block cardbody %}
+    {% infobox %}
+    <p class="mb-3">
+        This council’s climate plans as of 20th September 2021 were assessed and scored by trained Climate Emergency UK volunteers, as part of the Council Climate Plan Scorecards project.
+    </p>
+    <p class="mb-3">
+        A new scorecard based on council actions will be released in Autumn 2023. <a href="https://councilclimatescorecards.uk/methodology/">Read more about this</a>.
+    </p>
+
+    <p class="mb-0 d-flex flex-wrap">
+        <a href="https://councilclimatescorecards.uk/councils/{{ council.slug }}/"
+           class="d-flex align-items-center mr-4 font-weight-bold">Show the full Scorecard
+
+            {% include 'caps/icons/external-link.html' with classes='external-link ml-1' role='presentation' %}
+
+        </a>
+        <a href="https://councilclimatescorecards.uk/methodology/"
+           class="d-flex align-items-center mr-4 font-weight-bold">Read more about the scoring process
+
+            {% include 'caps/icons/external-link.html' with classes='external-link ml-1' role='presentation' %}
+
+        </a>
+    </p>
+    {% endinfobox %}
+
+    {% if scoring_score.weighted_total == 0 %}
+
+        {% include 'caps/icons/question-circle.html' with width='2rem' height='2rem' classes='mr-3 mr-lg-4 d-none d-sm-block flex-grow-0 flex-shrink-0 text-red' role='presentation' %}
+
+        <div>
+            <p class="mb-0">
+                {{ council.name }} scored <strong>zero points</strong> in this assessment. This is because the council either didn’t have any published plans at the time of the assessment, or <a href="https://councilclimatescorecards.uk/methodology/">the plans didn’t qualify</a>.
+            </p>
+        </div>
+    {% endif %}
+    <div class="row mt-3 mt-lg-4">
```

Comment:
It feels like the alignment of these in a wide window could do with some work:

<img width="791" alt="Screenshot 2023-07-04 at 15 25 56" src="https://github.com/mysociety/caps/assets/5310/bfc40d2f-3daf-4aa6-a28b-03b1ab89adbd">